internal/pkg/exec/dockerengine.go

```diff
@@ -264,6 +270,28 @@ func ValidatePlatform(platform *string) error {
 	return nil
 }
 
+// ValidateOS checks if the entered string is a Docker-buildable operating system.
+func ValidateOS(os *string) error {
+	if os == nil {
+		return nil
+	}
+	if aws.StringValue(os) != LinuxOS {
+		return fmt.Errorf("OS %s is invalid; %s: %s", aws.StringValue(os), english.PluralWord(len(validOperatingSystems), "the valid operating system is", "valid operating systems are"), english.WordSeries(validOperatingSystems, "and"))
+	}
+	return nil
+}
+
+// ValidateArch checks if the entered string is a Docker-buildable architecture.
+func ValidateArch(arch *string) error {
+	if arch == nil {
+		return nil
+	}
+	if aws.StringValue(arch) != Amd64Arch {
```

Comment:
same here

Reply:
Same as above-- hopefully we'll support ARM architectures in the future and can add those to the validArchitectures list.

Reply:
done!